### PR TITLE
Add page for documenting packages

### PR DESCRIPTION
**Pull request by Jan200101**
Package docs

Includes a basic explanation of what packages are and a short guide on how to manually install packages.

Unsure if the Mod Installation in the Manual Installation section should be updated, I personally think that mods should continue to be for core mods and anything requiring human intervention.

### CODE REVIEW

**Comment by Jan200101** [comment]
Its been agreed upon that mods are preferred for manual installations

(also please squash so I don't have to rename the commit :womp:)

**Review by GeckoEidechse — COMMENTED** [review]
FYI, for the page to show up in the sidebar you also need to update [docs/SUMMARY.md](https://github.com/R2Northstar/NorthstarWiki/blob/main/docs/SUMMARY.md?plain=1#L22) ^^

**EDIT:** I cannot delete my own review, wtf?

File: docs/using-northstar/packages.md
Comment (GeckoEidechse) [inline docs/using-northstar/packages.md:1]: FYI, for the page to show up in the sidebar you also need to update [docs/SUMMARY.md](https://github.com/R2Northstar/NorthstarWiki/blob/main/docs/SUMMARY.md?plain=1#L22) ^^

Comment (Jan200101) [reply docs/using-northstar/packages.md:1]: Didn't know that, thanks.

Is there any easy way of previewing the Wiki locally? I didn't find anything (though I also didn't look very far)

Comment (GeckoEidechse) [reply docs/using-northstar/packages.md:1]: Not really and it's super frustrating. The closest I found was [`mdBook`](https://github.com/rust-lang/mdBook) cause that also uses the `SUMMARY.md` but there's nothing I found that also uses the GitBook specific syntax for certain widgets like

https://github.com/R2Northstar/NorthstarWiki/blob/f96a1767613b853a6d8502d4345324e82be2e9b0/docs/installing-northstar/manual-installation.md?plain=1#L4

and a primary reason why I am considering moving wiki but the free hosting we get from GitBook is kinda hard to beat :P

File: docs/using-northstar/packages.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Packages
+
+Packages are community created mods with the [Thunderstore](https://northstar.thunderstore.io/) Package structure retained used for installation via Mod Managers. They are located in the `Titanfall2\R2Northstar\packages` folder and can be accessed via the `Mods` menu at the bottom of the main menu.
+
+## Manual installation
+
+1. Download a mod from Thunderstore
+2. Right click the relevant Zip and Extract All
```
Comment (GeckoEidechse) [inline docs/using-northstar/packages.md]: Formatting
```suggestion
2. Right click the relevant Zip and _Extract All_ \
```

File: docs/using-northstar/packages.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Packages
+
+Packages are community created mods with the [Thunderstore](https://northstar.thunderstore.io/) Package structure retained used for installation via Mod Managers. They are located in the `Titanfall2\R2Northstar\packages` folder and can be accessed via the `Mods` menu at the bottom of the main menu.
```
Comment (GeckoEidechse) [inline docs/using-northstar/packages.md]:
```suggestion
Packages are community created mods with the [Thunderstore](https://northstar.thunderstore.io/) package structure retained used for installation via Mod Managers. They are located in the `Titanfall2\R2Northstar\packages` folder and can be accessed via the `Mods` menu at the bottom of the main menu.
```

**Review by GeckoEidechse — COMMENTED** [review]
Two quick things I noticed on checking again now. Will commit myself then merge ^^

**Review by GeckoEidechse — APPROVED** [review]
Should be good now ^^

**Pull request merged**